bin/start_cn.sh

```diff
@@ -125,5 +125,5 @@ fi
 if [ ${RUN_DAEMON} -eq 1 ]; then
     nohup ${STARROCKS_HOME}/lib/starrocks_be --cn "$@" >> $LOG_DIR/cn.out 2>&1 </dev/null &
 else
-    ${STARROCKS_HOME}/lib/starrocks_be --cn "$@" >> $LOG_DIR/cn.out 2>&1 </dev/null
+    exec ${STARROCKS_HOME}/lib/starrocks_be --cn "$@" >> $LOG_DIR/cn.out 2>&1 </dev/null
```

Comment:
why need to add exec here? why start_be.sh needn't it?

Reply:
Make starrocks_be run as pid 1 in container (instead of child of bash) , then it will receive signal from docker.